### ISSUE TITLE
NTR: Arboviruses PCR or RT-PCR in CSF positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8066]
New term request Arboviruses PCR or RT-PCR in CSF positive (samant_101219224821): Cerebrospinal fluid polymerase chain reaction; RNA; DNA  UMLS: 0368699

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000549
- label: Positive CSF arbovirus nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of arbovus nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR..
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8066
- synonyms:
